### ISSUE TITLE
Alerts for new lists are not being sent

### ISSUE THREAD

**Issue by nickdos** [AtlasOfLivingAustralia/alerts#36]
Reported by @Tasilee - I've not got any for quite a while either but I can see that new lists have been added in the last couple of weeks.

(Nov 2020), no alerts emails (including new lists) were sent in Test environment, use this ticket for code updating.

**Comment by nickdos** [comment]
Here's the last list update I got:

-------------

From: Atlas alerts <atlas-alerts@ala.org.au>
Date: Sunday, 17 September 2017 at 9:30 pm
Subject: Species lists

Update - Species lists
New species lists have been added.
To view details of the datasets that have been added/changed, click here
To disable this alert or to manage your alerts, click here
Datasets update
Name	Link
wolli	click to details http://collections.ala.org.au/public/show/dr7947

To stop receiving alerts for Species lists, click here. 
To stop receiving all Atlas of Living Australia alerts, click here.
